### DISCUSSION TITLE
how to disable icon time in NextJS 14

### DISCUSSION THREAD

**Discussion by goodlythink** [mantinedev/mantine#6454] [Q&A]
i use TimeInput on NextJS 14
how to disable icon time

```
  <TimeInput
      label="Input label"
      description="Input description"
      placeholder="Input placeholder"
    />
```

![timeinput](https://github.com/mantinedev/mantine/assets/12431734/678c4065-3a7c-43ce-9aed-5c287aa666a3)

**Comment by rtivital** [comment]
What browser do you use?

**Comment by goodlythink** [comment]
Google Chrome in Windows

**Reply by rtivital** [reply]
Have you imported `import '@mantine/dates/styles.css';` in your application?

**Comment by goodlythink** [comment]
That's right, I didn't import it

thank you very much
